Client/Frontend/Browser/OpenInHelper.swift

```diff
@@ -127,18 +132,24 @@ class OpenPdfInHelper: NSObject, OpenInHelper, UIDocumentInteractionControllerDe
     fileprivate var filepath: URL?
 
     required init?(response: URLResponse) {
-        guard let MIMEType = response.mimeType, MIMEType == MimeType.PDF.rawValue && UIApplication.shared.canOpenURL(URL(string: "itms-books:")!),
+        guard let MIMEType = response.mimeType,
+            (MIMEType == MimeType.PDF.rawValue || MIMEType == MimeType.EPUB.rawValue) && UIApplication.shared.canOpenURL(URL(string: "itms-books:")!),
```

Comment:
I think we should be able to do `MimeType(response.mimeType) != nil` to see if it is a supported Mimetype